### ISSUE TITLE
Add Typescript support

### ISSUE THREAD

**Issue by Zenoo** [dominicegginton/hex-color-opacity#16]
Hi, the Typescript definitions are missing from this project.

I added them manually in my project with

```typescript
declare module "hex-color-opacity" {
  export default function opacity(hex: string, opacity: number): string
}
```

But it would be cool if it was provided from the package itself.

All you would need to do is add a `.d.ts` file in your package publication with something like this inside:

```typescript
export default function opacity(hex: string, opacity: number): string
```

**Comment by dominicegginton** [comment]
Type definitions added in [`hex-color-opacity@v0.4.0`](https://github.com/dominicegginton/hex-color-opacity/releases/tag/v0.4.0), thanks @Zenoo

**Comment by Zenoo** [comment]
You need to add a name to the `.d.ts` file though.  

`opacity.d.ts` would work!

**Comment by dominicegginton** [comment]
@Zenoo my bad. Should be fixed now in [`hex-color-opacity@v0.4.2`](https://github.com/dominicegginton/hex-color-opacity/releases/tag/v0.4.2)
